models/product-type/src/product-type/product-type-draft/presets/sample-data-goodstore/bedding-bundle.ts

```diff
@@ -0,0 +1,57 @@
+import { LocalizedString } from '@commercetools-test-data/commons';
+import {
+  AttributeDefinitionDraft,
+  AttributeLocalizableTextType,
+} from '../../../../index';
+import type { TProductTypeDraftBuilder } from '../../../types';
+import * as ProductTypeDraft from '../../index';
+
+const beddingBundle = (): TProductTypeDraftBuilder =>
+  ProductTypeDraft.presets
+    .empty()
+    .key('bedding-bundle')
+    .name('Bedding Bundle')
+    .description('Product type used to bundle bedding components')
+    .attributes([
+      AttributeDefinitionDraft.presets
+        .empty()
+        .name('product-description')
+        .type(AttributeLocalizableTextType.random())
```

Comment:
Majority of the work on all of these was reconciling the `type` from the related data and ensuring it was accurate. `type` was not included in the autogenerated presets